geolocation.py

```diff
@@ -137,15 +187,22 @@ def get_country_from_location(location_string):
         # Check different positions of the token
         for position in [-1, 0]:
             pieces = location_string.split(separator)
+
             token = pieces[position].strip()
+            if len(token) > 2:
```

Comment:
@kdobolyi, can you please put an explanatory comment here? It wasn't immediately obvious to me what is going on.